distribution/lib/Standard/Table/0.0.0-dev/src/Refined_Types/Numeric_Column.enso

```diff
@@ -0,0 +1,31 @@
+## A column that contains numeric values.
+type Numeric_Column
+    private Value column operations_implementation
+
+    ## GROUP Standard.Base.Operators
+       ICON operators
+       Computes the absolute value of each element in the column.
+
+       > Example
+         Compute the absolute value of values in a column.
+
+             import Standard.Examples
+
+             example_abs = Examples.decimal_column.abs
+    abs self -> Numeric_Column =
+        self.operations_implementation.abs self.column
+
+    ## GROUP Standard.Base.Operators
+       ICON operators
+       Computes the sign of each element in the column.
+
+       It will return -1, 0 or 1 depending on the sign of the value.
+
+       > Example
+         Compute the signum of values in a column.
+
+             import Standard.Examples
+
+             example_signum = Examples.decimal_column.signum
+    signum self -> Numeric_Column =
```

Comment:
- there are two unary operations in the `Numeric_Column` type
- don't we need some **binary operations** as well?
- only then the _type classes_ pattern shines...